### ISSUE TITLE
Why was the update reverted?

### ISSUE THREAD

**Issue by TheMaskedMan99** [haxball/haxball-issues#819]
Honestly, im my opinion this was the best update so far, since you managed to reduce the extrapolation feature that was being hardly abused to give advantage in the game and making the gameplay far way cancerous.

Why was it reverted? 50 was a perfect number to deal with the imput lag that was the main aim of the feature... but now it's back to 200 where people abuse from it...

**Comment by basro** [comment]
Many reasons,

The negative reception was very strong, I received a lot of complaints from a lot individuals from all over the world and the day is just starting. I don't want to deal with such backlash.

The other reason is that the limit is not cheat proof, people would have been able to bypass it by modifying the game scripts. I don't want to incentivize people to do that. And because of how strong the negative reception of this change was I suspect it would not have taken long before people started sharing detailed instructions on how to bypass it.

So basically, even though I'd like to limit it, I can't. I'm sorry.

**Comment by TheMaskedMan99** [comment]
Okay. I understand the problem. Sorry for my first reaction to the re-change.

Also, I think there were good reactions too. In that FM topic look Isco's, arguably the best DM in game, post: "i support Basro on this one, mainly because this game is cancer more than ever, especially when players are taking the ball into the corner and play it there all game long with their 200 extrapolation".

Do you think removing extrapolation would be an option to avoid that kind of modifying the scripts solution? Ignoring all the backlash from negative reviews, do you think it could be viable, without affecting the gameplay?

**Comment by polackangry** [comment]
The problem is that previously this option has been added .. I have been playing Flash for years and then for a long time on HTML without using any extrapolation. But then I saw how much this option improves in my game, first it was 20, then 50, 80, 100 and even more.

When basro suddenly added the limit to 50, I was not able to switch and probably would just finished playing.

Disabling this option is currently the end of a playing for a lot players. As Basro mentioned somewhere, the only solution is a time machine and not implementing extrapolation at all. : P

**Comment by meteus** [comment]
If you think extrapolation is giving players some advantage, then use yourself too. I still don't think it gives any advantage, even I think it gives disadvantage cuz higher = shaky gameplay. But yeah, I don't want to play with feeling that my character is like 500 kg or something. Thanks Basro for reverting the update. This is the most good decision. Now lots of player doesn't want to stop playing cuz our extrapolation is back. Peace!

**Comment by TheMaskedMan99** [comment]
It gives indeed. I just don't use because I don't want to feel the game "shaky" as you say,

**Comment by dakin1** [comment]
@basro Honestly, I don't know why you reverted the update. Simply, you're only one who is destroying Haxball even as owner. Yes, it may sound so annoying for you, but it's just truth. 
Have you ever played competitive Haxball? No, you're just owner who earns the money over game who you own. 
When you updated Haxball, I tried to play and it felt so smooth as fuck like I've never had that smooth in game. Why? Because extrapolation was limit to 50, no one couldn't do these tricks.
It reminded me like it's Haxball in Flash version in 2010/2011/2012. 
If you have ever listened to us, Haxball would have over 5k players almost every day (maybe doesn't make sense, but never mind).
Legendary players are quitting this game (Isco, aguero10, rkd, Falk, etc) because of these players who are playing with over 50 fucking extrapolations and these players ARE spending time PLAYING in corners which is annoying as fuck. You don't feel that pain like us who are trying to play normal gameplay. Unfortunanely, it's impossible because of these extrapolations. 
It would be good if you start following European leagues/community and maybe looking some records in leagues. 
These players have an such unfair advantage anywhere on the field (espically in corners) with over 100 extrapolations. How is it fair? For you it's fair because you never play the game and you still say it's only in our heads. No, it's not, you brainless dickhead. 
The only way to try save Haxball at least a bit is removing the extrapolation like we had HTML before extrapolation came out (I mean when HTML first came out, in that time it was smooth as fuck anywhere, every player could enjoyed it). 
Are you scared of these complains which you got by these cancer brainless players? Why you can't be like Fritz (of course, you don't know these players) who is best admin ever who doesn't care for other players and cared for the game.
If you don't do that, you may regret yourself and ask yourself "why these players aren't playing this game anymore". Of course, you don't care for other players, you CARE for cancer brainless trash corner players. And still earns money over these ads, well played.
Just remove the fucking extrapolation and make HTML like when HTML came out first before extrapolation and it will save Haxball at least a bit and more enjoyable.

#LetsMakeHaxballGreatAgain

**Comment by dakin1** [comment]
P.S. Maybe you should start to listen to these top teams, they will say same thing as I did in first post.
You'll get it. I really hope for you to not be dickhead this time and listen to these top teams.

**Comment by meteus** [comment]
@dakin1 wtf are you talking about you are saying ppl who uses extrapolation has an unfair advantage lmao then open yourself too. you just don't want to lose against very skilled players. 


"of course, you don't care for other players, you CARE for cancer brainless trash corner players."

I think you are non skilled worthless player thats why you crying like that. I bet you can't even do any tricks in corner, and get fucked every match. lol also no one gives any fuck to isco, aguero10 and any ppl. and don't talk bullshit about top teams. every top team players using more than 120 extrapolation.

**Comment by dakin1** [comment]
@meteus I'm wearing another nick, that's why I'm unknown. I've been playing this game since 2010, who are you?
You're headless chicken and brainless cancer player who doesn't even realise how Haxball was used to. I said, basro just destroyed the game putting the extrapolations. I've won 3 HCLs, FM 4/5 times, everywhere. 
You are just stupid and you did prove it right there, what a player.

**Comment by safsdkfjkl23mfdsa** [comment]
@dakin1 if you want a smooth gameplay then fuck off and play flash version.

the game is changed, accept it or not no one cares about your shit. and if you think you are good enough come 1v1 then we discuss stuff. stop thinking that you are good in team play. i don't give any fuck to team play. if you have balls come 1v1 in classic then and you meet with my corner skills.  you stupid faggot.

**Comment by akelaa** [comment]
@dakin1 keep crying you stupid. you don't even have balls to write your real nickname. cuz i didn't even won any hcl, any fm 

ahahahah pussy if you have balls come im inviting you lets 1v1 and meet with the god of the corner and ill end you up in 3-0 you fuckhead.
